plugin/trino-loki/src/main/java/io/trino/loki/LokiRecordCursor.java

```diff
@@ -84,8 +89,8 @@ private Object getEntryValue(int field)
         int columnIndex = fieldToColumnIndex[field];
         return switch (columnIndex) {
             // TODO: case 0 -> getSqlMapFromMap(columnHandles.get(columnIndex).columnType(), fields.labels());
-            case 1 -> entry.getTs();
-            case 2 -> entry.getLine();
+            case 0 -> entry.getTs();
```

Comment:
This is the same order as you devfine them in LokiMetadata right? I am wondering if there is a way to not to spread this knowledge across so many places.

Reply:
`fieldToColumnIndex` holds the ordinal position see line 51. So it's actually fixed albeit hard to reason about. This should be easier once we use the page provider instead.